tests/gar.test.mjs

```diff
@@ -1209,7 +1236,7 @@ describe('GatewayRegistry', async () => {
     const observerAddress = 'observer-address-'.padEnd(43, 'a');
 
     let gatewayMemory = sharedMemory;
-    before(async () => {
+    beforeEach(async () => {
```

Comment:
For some reason yes and I can't yet explain why. When I ran the integration tests as a whole, the before block would fail token supply invariance checks in Buy-Record. When I ran with --test-name-pattern on Save-Observations tests only, they would pass. When I ran all the tests in gar.test.mjs, it would fail again. I read over the code 3-4 times and couldn't explain it but I suspect it has to do with the time of when a nested before block runs. This shot in the dark change was the only way I could get it to reliably pass.